servers/src/main/java/tachyon/worker/block/evictor/LRUEvictor.java

```diff
@@ -99,20 +126,12 @@ public EvictionPlan freeSpace(long availableBytes, BlockStoreLocation location)
    */
   @Override
   public void onAccessBlock(long userId, long blockId) {
-    if (mLRUCache.containsKey(blockId)) {
-      mLRUCache.get(blockId);
-    } else {
-      mLRUCache.put(blockId, true);
-    }
+    mLRUCache.put(blockId, true);
```

Comment:
Is there a LinkedHashSet that could work for this purpose? Then, it wouldn't need the useless boolean.